Maple2.Server.Game/Model/Field/Actor/ActorStateComponent/MovementState.Tasks.cs

```diff
@@ -0,0 +1,322 @@
+﻿using Maple2.Model.Enum;
+using Maple2.Server.Game.Model.Enum;
+using Maple2.Server.Game.Model.Skill;
+using System.Numerics;
+using static Maple2.Server.Game.Model.Field.Actor.ActorStateComponent.TaskState;
+
+namespace Maple2.Server.Game.Model.Field.Actor.ActorStateComponent;
+
+public partial class MovementState {
```

Comment:
I feel like these tasks classes could live on their own files, this file is big and could get bigger?